pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -119,7 +119,7 @@ type JobStatus struct {
 	Progress float64 `json:"progress,omitempty"`
 
 	// Summary of processed actions
-	Summary []JobResourceSummary `json:"summary,omitempty"`
+	Summary []*JobResourceSummary `json:"summary,omitempty"`
```

Comment:
using a pointer does not change the output and simplifies sharing a summary as we iterate a list (now in a function rather than inline)